app/Http/Controllers/CommentsController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CommentFormRequest;
+
+use App\Comment;
+
+use Illuminate\Http\Request;
+
+class CommentsController extends Controller
+{
+    public function newComment(CommentFormRequest $request)
+    {
+        $comment = new Comment(array(
```

Comment:
```suggestion
        $comment = new Comment(array(
```
dùng `[]` thay cho `array()`